src/main/java/mate/academy/spring/model/dto/request/OrderRequestDto.java

```diff
@@ -0,0 +1,15 @@
+package mate.academy.spring.model.dto.request;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import lombok.Getter;
+import lombok.Setter;
+import mate.academy.spring.model.Ticket;
+
+@Getter
+@Setter
+public class OrderRequestDto {
+    private List<Ticket> tickets;
```

Comment:
If you need all data about each ticket and id wouldn't be enough, it is better to use dto object instead of entity